components/renderers/widgets/widgeta/widgeta-renderer.ts

```diff
@@ -0,0 +1,57 @@
+///<reference path="../../../references.ts"/>
+
+class WidgetARenderer implements JSONForms.IRenderer {
```

Comment:
What the difference between `WidgetARenderer` and `WidgetBRenderer`? Do we need this one at all?